testdata/project-v4-with-deploy-image/config/crd/patches/cainjection_in_busyboxes.yaml

```diff
@@ -3,5 +3,5 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    cert-manager.io/inject-ca-from: $(CERTIFICATE_NAMESPACE)/$(CERTIFICATE_NAME)
+    cert-manager.io/inject-ca-from: CERTIFICATE_NAMESPACE/CERTIFICATE_NAME
```

Comment:
Is this change relevant?

Reply:
I am not sure why it is not in the master branch. 
The v4 is using v2-alpha and then it should be in this away.

Reply:
See we should not have the `$(` (i.e https://github.com/kubernetes-sigs/kubebuilder/tree/master/testdata/project-v4-config/config/crd) but we have a bug in the master for the deploy image plugin (https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v3-with-deploy-image/config/crd/patches/cainjection_in_memcacheds.yaml#L6) which are probably fixing here: https://github.com/kubernetes-sigs/kubebuilder/pull/2985/files#diff-deedc95e5fb2c50078118e8e46f65c026e90b6d7670b7b46ee5f7f2b285814f4R288-R306

Reply:
@varshaprasad96 we have a bug in master, see: https://github.com/kubernetes-sigs/kubebuilder/pull/3104/files